templates/base.html.twig

```diff
@@ -14,7 +14,15 @@
 	<body>
 		{% block header %}
 			<header>
-					<img class="header--logo" src="{{ asset('build/images/Logo_1gscop.png') }}"/>
+
+				<div class="header--logo" >
+					<a href="{{ path("app_login") }}"><img class="logo" src="{{ asset('build/images/Logo_1gscop.png') }}"/></a>
+				</div>
+				{% block Admin %}{% endblock %}
+				{% if app.user %}
```

Comment:
if app.user should be replaced by the is_granted twig method with a specific role or a IS_AUTHENTICATED_FULLY